src/components/main-dashboard/DailyWeather.jsx

```diff
@@ -11,7 +11,8 @@ export default function DailyWeather() {
       const { list } = dailyWeather;
       const dailyWeatherInfo = list.map((weatherData, index) => {
         const weatherObj = {
-          id: index,
+          key: index,
+          id: weatherData.weather[0].id,
```

Comment:
iconKey key-value can be removed in weatherObj object since it is not destructured/utilized in `CreateDailyWeather`

Reply:
IconKey was renamed to icon in the latest change to allow for destructuring in `getIconName()`